scripts/bootstrap_assets.py

```
@@ -0,0 +1,489 @@
+
+    ensure_directory(destination.parent, dry_run=dry_run)
+
+    if dry_run:
+        LOGGER.info("DRY RUN: would download %s -> %s", url, destination)
+        return
+
+    tmp_destination = destination.with_name(destination.name + ".part")
+    if tmp_destination.exists():
+        tmp_destination.unlink()
+
+    request = Request(url, headers={"User-Agent": "lexnlp-bootstrap/1.0"})
```

Ruff (0.15.4)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)